xc/common/utils/prjxray_routing_import.py

```diff
@@ -999,17 +1001,62 @@ def get_pip_wire_names(pip_pkey):
     return get_pip_wire_names
 
 
+def check_duplicate_chan_pin_edges(
+        nodes_dict, node_mapping, src_graph_node, dest_graph_node, switch_pkey
+):
+    """
+    This function checks whether there are multiple edges between the same
+    (CHAN, PIN) pair with the same switch.
+
+    It may happen that a same CHAN <-> PIN edge is generated and this is unaccepted
+    by VPR, as it allows only multiple edges between CHAN nodes.
```

Comment:
The error you linked now accommodates multiple CHAN <-> PIN edges?  So this comment is wrong?

Reply:
Hmm, actually I think I found a bug in VPR, making this whole check useless. Testing it now